weave/__init__.py

```diff
@@ -2,16 +2,12 @@
 
 from weave import version
 
-from weave.trace.api import *
+from weave.trace import *
```

Comment:
nit: I think this re-exporting stuff is nice because it allows us to allow users to access symbols differently than their source location. However, import * seems not ideal for this case as we might accidentally re-export more symbols than is desirable here.

Reply:
I don't think that the author of trace.init should be able to control what gets exported here. Yes, __all__ would do that, but personally i think that moves the control to the wrong spot